apps/parse/lib/parse/alerts.ex

```diff
@@ -75,6 +76,11 @@ defmodule Parse.Alerts do
     copy(cause)
   end
 
+  defp cause(alert) do
+    Logger.error("No cause found in alert: #{inspect(alert)}")
```

Comment:
Naturally, now that I have added this, all the alerts are parsing. Still, I think it's a good guard instead of just failing.

Reply:
Do we want to serve alerts with empty string causes? With the existing setup for alerts though I don't see enforcing this contract as the API's job necessarily. So 👍